src/playlist.js

```diff
@@ -223,14 +223,14 @@ export const sumDurations = function(playlist, startIndex, endIndex) {
  *         point.
  * @function safeLiveIndex
  */
-export const safeLiveIndex = function(playlist) {
+export const safeLiveIndex = function(playlist, suggestedPresentationDelay) {
   if (!playlist.segments.length) {
     return 0;
   }
 
   let i = playlist.segments.length - 1;
   let distanceFromEnd = playlist.segments[i].duration || playlist.targetDuration;
-  const safeDistance = distanceFromEnd + playlist.targetDuration * 2;
+  const safeDistance = suggestedPresentationDelay || distanceFromEnd + playlist.targetDuration * 2;
```

Comment:
Super minor, but would it be clearer to have parentheses around the second part of the `||`?